cellpack/tests/test_db_recipe_handler.py

```diff
@@ -0,0 +1,169 @@
+from cellpack.autopack.DBRecipeHandler import DBRecipeHandler
+from cellpack.tests.mocks.mock_db import MockDB
+
+mock_db = MockDB({})
+
+
+def test_is_nested_list():
+    assert DBRecipeHandler.is_nested_list([]) is False
+    assert DBRecipeHandler.is_nested_list([[], []]) is True
+    assert DBRecipeHandler.is_nested_list([[1, 2], [3, 4]]) is True
```

Comment:
Possible edge case:
```
[ 1, [1,2,3]]
```
Should this count as a nested list?

Reply:
Thanks! you are right, it failed with this edge case. This function is mainly checking value's for `bounding_box`, `representation` and `position`. We could make it more robust for future use